--- xpuzzles-7.0.1/barrel/xbarrel.man~	2004-07-18 23:57:08.000000000 +0000
+++ xpuzzles-7.0.1/barrel/xbarrel.man	2004-07-18 23:57:08.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xbarrel \- Barrel X widget
 .SH SYNOPSIS
-.B /usr/games/xbarrel
+.B /usr/bin/xbarrel
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/cubes/xcubes.man~	2004-07-18 23:57:11.000000000 +0000
+++ xpuzzles-7.0.1/cubes/xcubes.man	2004-07-18 23:57:11.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xcubes \- Cubes X widget
 .SH SYNOPSIS
-.B /usr/games/xcubes
+.B /usr/bin/xcubes
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/dino/xdino.man~	2004-07-18 23:57:16.000000000 +0000
+++ xpuzzles-7.0.1/dino/xdino.man	2004-07-18 23:57:16.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xdino \- Dino X widgets
 .SH SYNOPSIS
-.B /usr/games/xdino
+.B /usr/bin/xdino
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/hexagons/xhexagons.man~	2004-07-18 23:57:19.000000000 +0000
+++ xpuzzles-7.0.1/hexagons/xhexagons.man	2004-07-18 23:57:19.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xhexagons \- Hexagons X widget
 .SH SYNOPSIS
-.B /usr/games/xhexagons
+.B /usr/bin/xhexagons
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/mball/xmball.man~	2004-07-18 23:57:21.000000000 +0000
+++ xpuzzles-7.0.1/mball/xmball.man	2004-07-18 23:57:21.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xmball \- Masterball X widget
 .SH SYNOPSIS
-.B /usr/games/xmball
+.B /usr/bin/xmball
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/mlink/xmlink.man~	2004-07-18 23:57:22.000000000 +0000
+++ xpuzzles-7.0.1/mlink/xmlink.man	2004-07-18 23:57:22.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xmlink \- Missing Link X widget
 .SH SYNOPSIS
-.B /usr/games/xmlink
+.B /usr/bin/xmlink
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/oct/xoct.man~	2004-07-18 23:57:24.000000000 +0000
+++ xpuzzles-7.0.1/oct/xoct.man	2004-07-18 23:57:24.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xoct \- Octahedron X widget
 .SH SYNOPSIS
-.B /usr/games/xoct
+.B /usr/bin/xoct
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/panex/xpanex.man~	2004-07-18 23:57:26.000000000 +0000
+++ xpuzzles-7.0.1/panex/xpanex.man	2004-07-18 23:57:26.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xpanex \- Panex X widget
 .SH SYNOPSIS
-.B /usr/games/xpanex
+.B /usr/bin/xpanex
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/pyraminx/xpyraminx.man~	2004-07-18 23:57:27.000000000 +0000
+++ xpuzzles-7.0.1/pyraminx/xpyraminx.man	2004-07-18 23:57:27.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xpyraminx \- Pyramid X widget
 .SH SYNOPSIS
-.B /usr/games/xpyraminx
+.B /usr/bin/xpyraminx
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/rubik/xrubik.man~	2004-07-18 23:57:29.000000000 +0000
+++ xpuzzles-7.0.1/rubik/xrubik.man	2004-07-18 23:57:29.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xrubik \- Magic cube X widgets
 .SH SYNOPSIS
-.B /usr/games/xrubik
+.B /usr/bin/xrubik
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.0.1/skewb/xskewb.man~	2004-07-18 23:57:31.000000000 +0000
+++ xpuzzles-7.0.1/skewb/xskewb.man	2004-07-18 23:57:31.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xskewb \- Skewb X widgets
 .SH SYNOPSIS
-.B /usr/games/xskewb
+.B /usr/bin/xskewb
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
--- xpuzzles-7.4.2/threed/xthreed.man~  2008-04-24 21:59:26.000000000 +0200
+++ xpuzzles-7.4.2/threed/xthreed.man   2008-12-08 14:04:14.000000000 +0100
@@ -20,7 +20,7 @@
 .SH NAME
 xthreed \- Threed X widget
 .SH SYNOPSIS
-.B /usr/games/xthreed
+.B /usr/bin/xthreed
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-{foreground|fg} {color}]
 [-{background|bg} {color}] [-frame {color}
--- xpuzzles-7.0.1/triangles/xtriangles.man~	2004-07-18 23:57:34.000000000 +0000
+++ xpuzzles-7.0.1/triangles/xtriangles.man	2004-07-18 23:57:34.000000000 +0000
@@ -24,7 +24,7 @@
 .SH NAME
 xtriangles \- Triangles X widget
 .SH SYNOPSIS
-.B /usr/games/xtriangles
+.B /usr/bin/xtriangles
 [-geometry [{width}][x{height}][{+-}{xoff}[{+-}{yoff}]]]
 [-display [{host}]:[{vs}]] [-[no]mono] [-[no]{reverse|rv}]
 [-{foreground|fg} {color}] [-{background|bg} {color}]
